app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+
+  def new
+    @user = User.new
```

Comment:
You may not need this - it's more usual to use form_tag for a login form - you're not editing a model.